### PR TITLE
Add support for diarization_type, speakers_count, remove_atmospherics

### PR DESCRIPTION
**Pull request by kirillatrev**
Add support for diarization_type, speakers_count, remove_atmospherics

Follow up on 
https://github.com/revdotcom/revai-python-sdk/pull/104

### CODE REVIEW

File: src/rev_ai/apiclient.py [hunk 1]
```diff
@@ -137,6 +139,10 @@ def submit_job_url(
                                                    segments_to_transcribe, speaker_names,
                                                    source_config, notification_config,
                                                    skip_postprocessing,
+						   remove_atmospherics,
```
Comment (dmtrrk) [inline src/rev_ai/apiclient.py:142]: is this correct you added params except `diarization_type`?

Comment (kirillatrev) [inline src/rev_ai/apiclient.py]: It appears that they were declared on the prototype but were not actually submitted....

Comment (dmtrrk) [reply src/rev_ai/apiclient.py]: @eugenep-rev do you know anything about this?

I just can guess that was intentionally, but looks like a bug

**Comment by kirillatrev** [comment]
@hrist0stoichev hrist0stoichev thank you for your contribution.

**Review by eugenep-rev — APPROVED** [review]

**Pull request merged**